articles/sharing_settings.md

```diff
@@ -8,23 +8,37 @@ order: 2
+Synapse has two ways to control who can access your content. `Sharing Settings`, or permissions, determine _who_ can access content and at what level. `Conditions for Use` define _how_ users with access may use the data. All content in Synapse has Sharing Settings but not all content has Conditions for Use. See the [Conditions for Use](/articles/access_control.html) article for information about those settings. This article covers sharing settings only.
 
 <a name="sharing-setting"></a>
 
 ## Sharing Settings
 
-You can use the `Sharing Settings` to grant different levels of access to individuals or `Teams`. Often, users leverage these features to invite people to view or collaborate on a Synapse `Project`. Here are the kinds of access currently supported:
+You can use the `Sharing Settings` to grant different levels of access to individuals or `Teams`. Often, users leverage these features to invite people to view or collaborate on a Synapse `Project`. Sharing Settings can apply to an entire project or for individual files or folders within a project.
 
-* Can view
-* Can download
-* Can edit
-* Can edit & delete
-* Administrator
+There are two primary groups: **Public** and **Private**. The `Private` sharing setting limits access to only specified users and teams. By default, new projects are `Private`. The `Public` sharing setting applies to users on the web and any registered, logged-in Synapse user.
 
-`Sharing Settings` can apply to an entire project or for individual files or folders within a project.
+When sharing with specific users and teams, Synapse supports the following permissions: view, download, edit, edit/delete, and administrator.
 
-`Sharing Settings` have two primary groups: **Public** and **Private**. The `Private` sharing setting limits access to only specified users and  teams. By default, new projects are `Private`. The `Public` sharing setting applies to users on the web and any registered, logged-in Synapse user.
+### View permissions
+
+View permissions give a Synapse user the ability to see that something in Synapse exists (like a project, file, or folder). They can discover it using Synapse search, and it will be visible to them if included in a [view](articles/views.html). If there are [annotations](https://docs.synapse.org/articles/annotation_and_query.html) associated with it, they can see these as well. They cannot see the contents of a file, including if a preview of the file is available in the web. View permissions are the only permissions that can be granted to the public (anonymous users).
```

Comment:
Do project, file, folder here need to be capitalized?

Reply:
Yup! thanks!

Reply:
```suggestion
View permissions give a Synapse user the ability to see that something in Synapse exists (like a Project, File, or Folder). They can discover it using Synapse search, and it will be visible to them if included in a [View](articles/views.html). If there are [annotations](https://docs.synapse.org/articles/annotation_and_query.html) associated with it, they can see these as well. They cannot see the contents of a File, including if a preview of the file is available in the web. View permissions are the only permissions that can be granted to the public (anonymous users).
```